ext/afform/core/CRM/Afform/DAO/AfformSubmission.php

```diff
@@ -87,7 +87,7 @@ public function __construct() {
    *   Whether to return the plural version of the title.
    */
   public static function getEntityTitle($plural = FALSE) {
-    return $plural ? E::ts('Form Builder Submissions') : E::ts('Form Builder Submission');
+    return $plural ? E::ts('FormBuilder Submissions') : E::ts('FormBuilder Submission');
```

Comment:
@aydun this is actually an autogenerated file. Have to edit the xml for this file and then go to the afform core extension dir and run `civix generate:entity-boilerplate`

Reply:
Ah, I see the xml file was updated too. That's fine.